plugins/inputs/nightscout/README.md

```diff
@@ -0,0 +1,1142 @@
+# Nightscout Input Plugin
+
+The nightscout plugin collects blood glucose related telemtry from a nightscout site. Not all available data is collected at this time, simply the basics needed for active monitoring.
+
+### Configuration:
+
+```toml
+  ## This is a list of nightscout sites to pull data from.
+  ##  example: servers = ["https://my.nightscout.site/api/v1/entries"]
+  servers = []
+  ## The person is the owner of the nightscout site. The order of the list of owners must match the order of the
+  ## list of servers, for example the first server should match the first owner.
+  # owners = ""
+
+  ## The secret is the SHA-1 hash of the api secret used to access the site.
+  secret = ""
```

Comment:
I'd probably leave that commented out, to encourage people to use tokens.

Reply:
It needs both the token and the secret. I will better document this. The secret is the password for the site, everyone needs it for any access at all. The token is specific to a user and provides a small measure of RBAC.
@danielnelson

Reply:
You can get data from your site via curl with only token, same with OpenAPS scripts. The API secret is a master-key-root-level-access to everything, without any further ACLs, as far as I'm aware.

Reply:
Thanks for this, I made the changes and confirmed they work.